.github/workflows/dev-containers.yml

```diff
@@ -25,18 +25,40 @@ jobs:
       run: yarn type-check
     - name: Lint
       run: yarn lint
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v2
     - name: Package
-      run: yarn package
-    - name: TGZ name
+      run: yarn package && yarn package-standalone
```

Comment:
If I recall, the `cannot resolve t` warnings the minified dynamic require/import calls in `yargs` (`t` is the mangled name).

You're noticing some of the reasons I split things across this PR and https://github.com/devcontainers/cli/pull/385 :slightly_smiling_face:.

#385 compiles the binaries on each target OS, because I realized `@vercel/pkg` wasn't cross-compiling the native modules from linux ([their docs](https://github.com/vercel/pkg#native-addons) seem ambiguous at best). I thought they may have included logic to re-run `node-gyp` to recompile a native module (kinda similar to electron's packager).

I wasn't sure how they'd do that without the target's compilers, but I'll admit I'm ignorant of the current state of cross-compiling from linux to windows/macos.